tests/functional/oauth_query_param_validation.js

```diff
@@ -349,35 +349,6 @@ define([
           client_id: UNTRUSTED_CLIENT_ID,
           scope: UNTRUSTED_SCOPE
         }));
```

Comment:
There is also a `  'verification_redirect': STRING_TYPE.optional()` in post-metrics-errors, but do we leave it??